### PR TITLE
End two AB tests that served their time. Add analytics for source of check.

### PR DESCRIPTION
**Pull request by lagerros**
Showing opinions seemed to help. Unclear effect of showing recommendations on frontpage (but I have a prior, and like the feature, so don't want to remove yet).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206082808839622) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/dialogues/DialoguesList.tsx [hunk 1]
```diff
@@ -323,7 +320,7 @@ const DialoguesList = ({ classes }: { classes: ClassesType<typeof styles> }) =>
             <DialogueMatchRow key={index} rowProps={rowProps} classes={classes} showMatchNote={true} />
           ))}
           {showReciprocityRecommendations && currentUser?.showRecommendedPartners && recommendedDialoguePartnersRowPropsList?.map((rowProps, index) => (
-            !rowProps.hideInRecommendations && <DialogueRecommendationRow key={index} targetUser={rowProps.targetUser} checkId={rowProps.checkId} userIsChecked={rowProps.userIsChecked} userIsMatched={rowProps.userIsMatched} showSuggestedTopics={showTopics} onHide={hideRecommendation} />
+            !rowProps.hideInRecommendations && <DialogueRecommendationRow key={index} targetUser={rowProps.targetUser} checkId={rowProps.checkId} userIsChecked={rowProps.userIsChecked} userIsMatched={rowProps.userIsMatched} showSuggestedTopics={true} onHide={hideRecommendation} />
```
Comment (darkruby501) [inline packages/lesswrong/components/dialogues/DialoguesList.tsx:323]: The presence of a prop implicitly without value assignment implicitly means its true, so you don't have to say ={true}

```suggestion
            !rowProps.hideInRecommendations && <DialogueRecommendationRow key={index} targetUser={rowProps.targetUser} checkId={rowProps.checkId} userIsChecked={rowProps.userIsChecked} userIsMatched={rowProps.userIsMatched} showSuggestedTopics onHide={hideRecommendation} />
```

**Review by darkruby501 — APPROVED** [review]
Looks good except for minor change around setting prop value to true

File: packages/lesswrong/components/users/DialogueMatchingPage.tsx [hunk 2]
```diff
@@ -952,6 +952,8 @@ const DialogueCheckBox: React.FC<{
     if (!currentUser) return;
 
     const response = await upsertUserDialogueCheck({ targetUserId, checked: event.target.checked, checkId });
+
+    captureEvent("newDialogueCheck", {checked: response.data.upsertUserDialogueCheck.checked})
```
Comment (darkruby501) [inline packages/lesswrong/components/users/DialogueMatchingPage.tsx:956]: I would extract `checked` and reuse in the different places.

So:

const checked = event.target.checked

And then use that in both the upsert call and the capture event

Comment (lagerros) [reply packages/lesswrong/components/users/DialogueMatchingPage.tsx:956]: yes that's a nice point. feel free to make that commit. 

Personally I really just want to get this shipped now, it's been many days. can clean up and make nicer in future. merging.

**Pull request merged**